Makefile

```diff
@@ -12,14 +14,14 @@ endif
 
 .PHONY: run
 run: build
-	@docker run agrdocker/agr_schemas:latest
+	@docker run agrdocker/agr_schemas:${DOCKER_BUILD_TAG}
```

Comment:
Switch out `agrdocker` for ${REG} ?

Reply:
As the container is only built for running the schemaPR pipeline, and is not intended to be pushed, I'll change it to `agrlocal` instead